openapi/openapi.go

```diff
@@ -28,7 +32,11 @@ func LoadOpenAPI(openAPIFile string) (*OpenAPI, error) {
 	if err != nil {
 		return nil, fmt.Errorf("unable to read file \"%s\": %w", openAPIFile, err)
 	}
-	yaml.Unmarshal(content, &api)
+
+	err = yaml.Unmarshal(content, &api)
+	if err != nil {
+		return nil, err
```

Comment:
Please wrap this with a unique error message specific to this codepath for easy reference if this ever needs to be troubleshooted

Reply:
sorted tjom